### DISCUSSION TITLE
Difficulties making a mapped-file data frame

### DISCUSSION THREAD

**Discussion by bosireerick** [felixgrunberger/microbepore#2] [Q&A]
Hi Felix,
I am learning how to analyze nanopore RNAseq data and I am trying your mapped read analysis code. I have difficulties with getting the mapped-read data frame. When I run the code with my data it returns zero rows and columns. And consequently the saved mapped-data tsv file is empty. Do you have an idea of what I might need to do? I can share some of the files if you might have a chance to look at them. Thanks in advance.
Erick.

**Comment by felixgrunberger** [comment]
I suggest that you run the read_bam_files function line by line to find the problem. What you need is a .gff3 file and the 2 bam files from mapping and remapping. If this does not work you can send me some files and I'll have a look into it.

**Comment by bosireerick** [comment]
Thanks Felix for looking at this. I used a .gff file that I downloaded from NCBI but it wasn't a .gff3. Could this be a problem? How do you generate a .gff3 file?

**Reply by felixgrunberger** [reply]
As far as I know, all gff files currently provided by NCBI are in gff3 format, so I assume that´s not the problem. Read more about the file formats and the columns required here: https://www.ncbi.nlm.nih.gov/genbank/genomes_gff/

**Comment by bosireerick** [comment]
Thanks a lot. I might be facing some other difficulties. Here are the preliminary files that I am using to to develop the pipeline:https://cornell.box.com/s/mlhhg9kvc96njf7s5q1o6j4cvt41n8jy.  Please try running them and see if they work. I greatly appreciate your help. 

Let me know if the link brings you to the folder. 
Best regards,
Erick

**Reply by felixgrunberger** [reply]
Sorry, but I don't have access to the folder.

**Comment by bosireerick** [comment]
Sorry, I needed to change some setting on the link. Please try this one: https://cornell.box.com/s/mlhhg9kvc96njf7s5q1o6j4cvt41n8jy. if it doesn't work, I will put them on google drive.  Thanks.

**Reply by felixgrunberger** [reply]
You can find the mapped read frame here: https://drive.google.com/drive/folders/1RO1yIAXWSnKgfe4-XuYFUCioHnHHLrpC?usp=sharing

I had to make some minor adjustments to the code (two script file are also in the drive folder), which ensure that the file names are handled correctly.

**Reply by bosireerick** [reply]
Thanks a lot Felix. I was able to run the code and get a data frame. I greatly appreciate your help. Erick
